tests/helpers.py

```diff
@@ -2330,3 +2333,646 @@ def retrieve_default_ingress_crt():
 
     with open(constants.DEFAULT_INGRESS_CRT_LOCAL_PATH, 'w') as crtfile:
         crtfile.write(decoded_crt)
+
+
+def verify_s3_object_integrity(original_object_path, result_object_path, awscli_pod):
```

Comment:
please don't place these functions in helpers.py but in the corresponding file under ocs_ci/ocs, like ocs_ci/ocs/utils.py or create a new ocs_ci/ocs/object.py or ocs_ci/ocs/s3.py

Reply:
Placing them in `utils.py` lead to an import loop, so I created a new helpers file 👍

Reply:
+1